client/web/compose/src/views/Admin/Modules/Edit.vue

```diff
@@ -216,8 +216,7 @@
 
                 <b-row no-gutters>
                   <c-form-table-wrapper
-                    :list-title="$t('edit.manageRecordFields')"
-                    :action-label="$t('edit.newField')"
+                    hide-add-button
```

Comment:
Dont you need the button label here?

Reply:
Nope, since we are hiding the add button here.

Reply:
Move system fields to a separate table (not wrapped)
Keep button text the same